ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala

```diff
@@ -4,8 +4,10 @@
 package akka.stream.alpakka.ftp
 
 import akka.stream.alpakka.ftp.FtpCredentials.AnonFtpCredentials
+
```

Comment:
which is ok ;) space between akka deps and language deps is something some people like; I don't mind